src/Pim/Bundle/EnrichBundle/Normalizer/CompletenessCollectionNormalizer.php

```diff
@@ -67,58 +117,77 @@ public function supportsNormalization($data, $format = null)
     }
 
     /**
-     * @param AttributeInterface $attribute
-     * @param array              $locales
+     * Returns how much completenesses have a ratio of 100 for a provided list of completeness.
      *
-     * @return array
+     * @param CompletenessInterface[] $completenesses
+     *
+     * @return int
      */
-    protected function normalizeAttributeLabels(AttributeInterface $attribute, array $locales)
+    protected function countComplete(array $completenesses)
     {
-        $labels = [];
-        foreach ($locales as $locale) {
-            $labels[$locale] = $attribute->getTranslation($locale)->getLabel();
+        $complete = 0;
+
+        foreach ($completenesses as $completeness) {
+            if (100 === $completeness->getRatio()) {
+                $complete++;
+            }
         }
 
-        return $labels;
+        return $complete;
     }
 
     /**
      * Returns the normalized channel completeness
      *
-     * @param string $format
-     * @param array  $context
-     * @param array  $channels
-     * @param array  $locales
+     * @param CompletenessInterface[] $completenesses
+     * @param string                  $format
+     * @param array                   $context
+     * @param array                   $localeCodes
      *
      * @return array
      */
-    protected function normalizeChannelCompleteness($format, array $context, array $channels, array $locales)
-    {
+    protected function normalizeChannelCompletenesses(
+        array $completenesses,
+        $format,
+        array $context,
+        array $localeCodes
+    ) {
         $normalizedCompChannels = [];
 
         //TODO: workaround in order to handle behat empty completeness
```

Comment:
I always create an empty array before adding values to it. I think it is just good practice.